formula/temporal_tables.rb

```diff
@@ -0,0 +1,37 @@
+class TemporalTables < Formula
+  desc "Temporal Tables PostgreSQL Extension"
+  homepage "http://pgxn.org/dist/temporal_tables/"
+  url "https://github.com/arkhipov/temporal_tables/archive/v1.1.0.tar.gz"
+  sha256 "1fe210a349d1418d097f229c36f30a1daef1ff17cf0f027685171c52e366308a"
+
+  depends_on :postgresql
```

Comment:
Fixed, thanks. Please note that there are several other formulas in core that use `:postgresql`:

```
Formula/check_postgres.rb:17:  depends_on :postgresql
Formula/ephemeralpg.rb:15:  depends_on :postgresql
Formula/eralchemy.rb:20:  depends_on :postgresql => :optional
Formula/freediameter.rb:25:    depends_on :postgresql
Formula/gearman.rb:26:  depends_on :postgresql => :optional
Formula/kitchen-sync.rb:21:  depends_on :postgresql => :optional
Formula/libpqxx.rb:16:  depends_on :postgresql
Formula/libzdb.rb:16:  depends_on :postgresql => :recommended
Formula/osm2pgrouting.rb:21:  depends_on :postgresql
Formula/osm2pgsql.rb:16:  depends_on :postgresql
Formula/pdns.rb:32:  depends_on :postgresql => :optional
Formula/pex.rb:15:  depends_on :postgresql
Formula/pg_top.rb:15:  depends_on :postgresql
Formula/pgcli.rb:16:  depends_on :postgresql
Formula/pgloader.rb:17:  depends_on :postgresql => :recommended
Formula/pgpool-ii.rb:14:  depends_on :postgresql
Formula/psqlodbc.rb:23:  depends_on :postgresql
Formula/pure-ftpd.rb:18:  depends_on :postgresql => :optional
Formula/qt5.rb:35:  depends_on :postgresql => :optional
Formula/sphinx.rb:25:  depends_on :postgresql => :optional
Formula/sysbench.rb:20:  depends_on :postgresql => :optional
Formula/urweb.rb:19:  depends_on :postgresql => :optional
Formula/zabbix.rb:20:    depends_on :postgresql if build.without? "mysql"
```